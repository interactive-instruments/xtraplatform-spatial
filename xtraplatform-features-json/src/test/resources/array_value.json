{
  "type" : "Feature",
  "id": "24",
  "properties" : {
    "erfasser_array": ["John Doe", "Jane Doe"],
    "kennung" : "611320001-1"
  }
}