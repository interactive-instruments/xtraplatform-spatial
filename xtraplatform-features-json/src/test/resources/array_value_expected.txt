START: -1 -1
[]{
    [id]: []:24
   >[id]
    [geometry, type]: []:Point
   >[geometry, type]
    [geometry, coordinates]: []:8.18523495507722
   >[geometry, coordinates]
    [geometry, coordinates]: []:49.698295103021096
   >[geometry, coordinates]
    [erfasser_array]: [1]:John Doe
   >[erfasser_array]
    [erfasser_array]: [2]:Jane Doe
   >[erfasser_array]
    [kennung]: [1]:611320001-1
   >[kennung]
}
END
